### PR TITLE
add json schema and status list

### PR DESCRIPTION
**Pull request by decentralgabe**
update formatting, add links, add json schema

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,6 +107,7 @@ The following labels should exist in all relevant repos
 | `w3c-vc-dm-2.0`      | `#01FF70` | W3C Verifiable Credential Data Model 2.0   |
 | `sd-jwt`             | `#85144B` | SD-JWT / SD-JWT-VC                         |
 | `pd-v2`              | `#F9A602` | Presentation Definition V2                 |
+| `sd-jwt`             | `#C86F42` | VC JSON Schema                             |
```
Comment (decentralgabe) [inline README.md:110]:
```suggestion
| `vc-json-schema`             | `#C86F42` | VC JSON Schema                             |
```

File: README.md [hunk 2]
```diff
@@ -106,6 +107,7 @@ The following labels should exist in all relevant repos
 | `w3c-vc-dm-2.0`      | `#01FF70` | W3C Verifiable Credential Data Model 2.0   |
 | `sd-jwt`             | `#85144B` | SD-JWT / SD-JWT-VC                         |
 | `pd-v2`              | `#F9A602` | Presentation Definition V2                 |
+| `vc-json-schema`             | `#C86F42` | VC JSON Schema                             |
```
Comment (decentralgabe) [inline README.md:110]:
```suggestion
| `vc-json-schema`     | `#C86F42` | VC JSON Schema                             |
```

File: README.md [hunk 3]
```diff
@@ -106,6 +107,7 @@ The following labels should exist in all relevant repos
 | `w3c-vc-dm-2.0`      | `#01FF70` | W3C Verifiable Credential Data Model 2.0   |
 | `sd-jwt`             | `#85144B` | SD-JWT / SD-JWT-VC                         |
 | `pd-v2`              | `#F9A602` | Presentation Definition V2                 |
+| `vc-json-schema`     | `#C86F42` | VC JSON Schema                             |
```
[the remark this answers is not in the document]
Comment (decentralgabe) [reply README.md:110]: don't think so - it needs to be done as a part of credential verification/validation wherever that is done

File: README.md [hunk 4]
```diff
@@ -274,96 +276,105 @@ Further, the key manager interface **must** be passed as an argument to _all_ pu
...
 > `did:ion` is included because it has been implemented in both Kotlin and Typescript. I'll be creating a Github issue soon to discuss when we think it makes sense to remove ION support from both SDKs
 
-### DID Document & Resolution Validation
+### [DID Document](https://www.w3.org/TR/did-core/) & [Resolution Validation](https://w3c-ccg.github.io/did-resolution/)
 
 | Feature      | Typescript | Kotlin | Rust | Swift |
 | ------------ | ---------- | ------ | ---- | ----- |
-| JSON Schema  | ❌          | ❌      | ❌    | ❌     |
-| Common Error | ❌          | ❌      | ❌    | ❌     |
+| JSON Schema  | ❌         | ❌     | ❌    | ❌   |
```
Comment (decentralgabe) [inline README.md:323]:
```suggestion
```

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (mistermoe) [reply README.md:110]: i imagine the JSON schema would live in 1 place and get used across all the SDKs

[the remark this answers is not in the document]
Comment (decentralgabe) [reply README.md:110]: this refers to the capabilities defined by the spec -- https://w3c.github.io/vc-json-schema/

it is more than single static definition of schemas, it's the ability to construct, resolve, and validate based on the spec.

we cannot assume that all of tbDEX VC schemas will be statically defined, we'll need to be able to encounter VCs using the spec and validate those

**Comment by decentralgabe** [comment]
added status

**Review by mistermoe — APPROVED** [review]

**Pull request merged**